330 quickstart features docs (#356)
conda/conda-pypi: docs/features.md

# Features

`conda-pypi` uses the `conda` plugin system to implement several features
that better improve the `conda` integration with the PyPI ecosystem. This
page is divided into basic and advanced sections to help you discover
which features are best for you.

- **Basic**: For users who just want to use conda and wheels packages
  together with no changes to their overall workflow.

> - **Advanced**: For users who want to experiment with conda and wheels

Reviewer: Nice, I like how you split this in to basic and advanced features.

  and work with cutting-edge plugin features.

## Basic features

(community-wheel-channel)=

> ### The community wheel channel

Reviewer 2: ```suggestion
### The conda-pypi channel
```

> The `conda-pypi` community wheel channel is a public channel on anaconda.org

Reviewer 2: ```suggestion
The `conda-pypi` channel is a public channel hosted by Anaconda
```

It'll be at https://repo.anaconda.cloud/conda-pypi, not anaconda.org.

Author: FYI, @dashagurova.

that makes pure Python packages from PyPI available through `conda install`.
After you add this channel, conda's solver can find and install these packages
alongside your regular conda packages in a single step.

> For instructions on how to set up the `conda-pypi` community wheel channel, see

Reviewer 2: ```suggestion
For instructions on how to set up the `conda-pypi` channel, see
```

the {doc}`quickstart`.

:::{admonition} Limitations
:class: warning

> The community wheel channel is designed to supplement existing conda channels,

Reviewer 2: ```suggestion
The conda-pypi channel is designed to supplement existing conda channels,
```

conda pypi install -e ./package1/ -e ./package2/
```

> ### `conda env` integrations

Reviewer: Hi @kathatherine, since this isn't currently supported and because having pip sections in environment.yaml files may not be needed at all going forward if users use this channel, I think we can remove this whole section.

Author: Can do!
